doc/services/shell/index.rst

```diff
@@ -74,6 +74,28 @@ procedure:
   to the shell.
 
 
+Telnet Backend
+==============
+
+Enabling :kconfig:option:`CONFIG_SHELL_BACKEND_TELNET` will allow users to use telnet
+as a shell backend. Connecting to it can be done using PuTTY or any ``telnet`` client.
+For example:
+
+.. code-block:: none
+
+  telnet <ip address> <port>
+
+By default the telnet client won't handle telnet commands. Although command support
```

Comment:
Suggestions: It might be helpful to clarify
```suggestion
By default the telnet client won't handle telnet commands and configuration. Although command support
```

Reply:
Sure